code/game/machinery/constructable_frame.dm

```diff
@@ -148,14 +150,14 @@
 						update_desc() // sets the description based on req_components
 						to_chat(user, desc)
 					else
-						to_chat(user, "<span class='warning'>This frame does not accept circuit boards of this type!</span>")
+						to_chat(user, "<span class='warning'>This frame does not accept [B.board_type] circuit boards!</span>")
```

Comment:
``This frame does not accept this kind of circuit boards``

I suggest "non-standard" as a replacement for "other" over in setup.dm's defines. It's used by air alarms, ai cores, fire locks, mecha, etc. A more indepth solution would be to add new machine type defines for them, but it's no big deal (and it may affect unit tests, since OTHER is checked there!)

Optional suggestion